docs/wiki/99_Developers/Developer_Instrument_Guide.md

```diff
@@ -0,0 +1,188 @@
+# Developer's Instrument Guide
```

Comment:
incorrect location https://github.com/aces/Loris/tree/main/docs/wiki/01_STUDY_PARAMETERS_SETUP/02_Clinical_Instruments

the placeholder clearly states "everything about instruments"